stellarphot/settings/tests/test_views.py

```diff
@@ -0,0 +1,68 @@
+from stellarphot.settings import Camera, ui_generator
+from stellarphot.settings.tests.test_models import TEST_CAMERA_VALUES
+
+
+class TestUiGenerator:
+    def test_camera(self):
+        ui = ui_generator(Camera)
+        # The description should be the beginning of the docstring
+        assert Camera.__doc__.strip().startswith(ui.description.split()[0])
+
+        # We always want to show nullable fields
+        assert ui.show_null
+
+        # Which means we don't need the button to show/hide them
+        assert ui.bn_shownull.layout.display == "none"
+
+        # For now we do not show the validaation output because it is painfully
```

Comment:
Typo of "validation", somewhat ironic since it passed initial review by the author....